src/uris/deepLinks/deepLink.ts

```diff
@@ -107,9 +111,22 @@ export const enum DeepLinkServiceAction {
 	TargetFetched,
 }
 
+export enum DeepLinkRepoOpenAction {
+	OpenInCurrentWindow = 'Open in Current Window',
+	OpenInNewWindow = 'Open in New Window',
+	AddToWorkspace = 'Add to Workspace',
+	Cancel = 'Cancel',
+}
+
+export const deepLinkRepoOpenActionToOpenWorkspaceLocation = {
+	[DeepLinkRepoOpenAction.OpenInCurrentWindow]: OpenWorkspaceLocation.CurrentWindow,
+	[DeepLinkRepoOpenAction.OpenInNewWindow]: OpenWorkspaceLocation.NewWindow,
+	[DeepLinkRepoOpenAction.AddToWorkspace]: OpenWorkspaceLocation.AddToWorkspace,
+};
+
 export interface DeepLinkServiceContext {
 	state: DeepLinkServiceState;
-	uri?: Uri | undefined;
+	uri?: string | undefined;
```

Comment:
Rename to `url` since it is a string (try to keep `uri` for `Uri` instances only)

Reply:
Done in commit: [Improvements from feedback](https://github.com/gitkraken/vscode-gitlens/pull/2503/commits/9e01ae826ec49b794057aa9570ec3df54ef0042d)